home/views.py

```diff
@@ -116,20 +121,67 @@ def error_404(request, *args, **argv):
         data = {}
         return render(request,'404.html', data)
 
+
+def send_email_to_user(otp,email):
+    import smtplib
+    con = smtplib.SMTP("smtp.gmail.com",587)
+    con.ehlo()
+    con.starttls()
+    admin_email = ""
+    admin_password = ""
+    con.login(admin_email,admin_password)
```

Comment:
Don't forgot to enter your email and password /
and make sure you enable less secure app on google account